settings.gradle

```diff
@@ -1 +1,3 @@
 rootProject.name = 'graphql-java-servlet'
+include 'graphql-java-common-http',
```

Comment:
Perhaps `graphql-http` as a better package name. Then the modules could be named `graphql-http-common` and `graphql-http-servlet`.